tests/test_r2s.py

```diff
@@ -388,3 +390,68 @@ def test_irradiation_setup_unstructured_nondef_tag():
     for nf, nft in izip(n_flux, n_flux_total):
         assert_array_equal(nf, fluxes[i])
         i+=1
+
+def _r2s_tests(r2s_run_dir):
+    os.chdir(thisdir)
+    # copy ../scripts/r2s.py to r2s_run_dir/r2s.py
+    os.chdir("..")
+    folderpath = os.getcwd()
+    dst = os.path.join(r2s_run_dir, "r2s.py")
+    copyfile(os.path.join(folderpath, "scripts", "r2s.py"), dst)
+
+    # run r2s step1 and step2
+    os.chdir(r2s_run_dir)
+    os.system('python r2s.py step1')
+    os.system('python r2s.py step2')
+
+    # output files of r2s step1
+    alara_inp = os.path.join(r2s_run_dir, "alara_inp")
+    alara_matlib = os.path.join(r2s_run_dir, "alara_matlib")
+    alara_fluxin = os.path.join(r2s_run_dir, "alara_fluxin")
+    blank_mesh = os.path.join(r2s_run_dir, "blank_mesh.h5m")
+    step1_file = os.path.join(r2s_run_dir, "r2s_step1.h5m")
+
+    exp_alara_inp = os.path.join(r2s_run_dir, "exp_alara_inp")
+    exp_alara_matlib = os.path.join(r2s_run_dir, "exp_alara_matlib")
+    exp_alara_fluxin = os.path.join(r2s_run_dir, "exp_alara_fluxin")
+    
+    # output files of r2s step2
+    e_bounds = os.path.join(r2s_run_dir, "e_bounds")
+    p_src = os.path.join(r2s_run_dir, "phtn_src.h5")
+    t_p_src = os.path.join(r2s_run_dir, "total_photon_source_intensities.txt")
+    src_c1 = os.path.join(r2s_run_dir, "source_1.h5m")
+  
+    exp_e_bounds = os.path.join(r2s_run_dir, "exp_e_bounds")
+    exp_t_p_src = os.path.join(r2s_run_dir, "exp_total_photon_source_intensities.txt")
+    
+    f1 = filecmp.cmp(alara_inp, exp_alara_inp)
+    f2 = filecmp.cmp(alara_matlib, exp_alara_matlib)
+    f3 = filecmp.cmp(alara_fluxin, exp_alara_fluxin)
+    f4 = filecmp.cmp(e_bounds, exp_e_bounds)
+    f5 = filecmp.cmp(t_p_src, exp_t_p_src)
+
+    os.remove(alara_inp)
+    os.remove(alara_fluxin)
+    os.remove(alara_matlib)
+    os.remove(blank_mesh)
+    os.remove(step1_file)
+    os.remove(e_bounds)
+    os.remove(p_src)
+    os.remove(t_p_src)
+    os.remove(src_c1)
+    os.remove(dst)
+    
+    assert_equal(f1, True)
+    assert_equal(f2, True)
+    assert_equal(f3, True)
+    assert_equal(f4, True)
+    assert_equal(f5, True)
+
+
+def test_r2s_script():
+    # test voxel r2s
```

Comment:
add a `try` block to import dagmc and if it fails, skip the test  - see `test_partisn.py` for an example